wgsd.go

```diff
@@ -63,19 +70,26 @@ func (p *WGSD) ServeDNS(ctx context.Context, w dns.ResponseWriter,
 	if err != nil {
 		return dns.RcodeServerFailure, err
 	}
-	if len(device.Peers) == 0 {
-		return nxDomain(p.zone, w, r)
-	}
 
 	// setup our reply message
 	m := new(dns.Msg)
 	m.SetReply(r)
 	m.Authoritative = true
 
+	allPeers := []host{
```

Comment:
Maybe call this type `peer` or use wgtypes.Peer and construct one for the CoreDNS host.

Will need to include AllowedIPs, which we can set to addresses obtained from the WireGuard interface.